.github/actions/publish-nuget/action.yml

```diff
@@ -31,6 +31,12 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - name: Setup Scripts
+      shell: pwsh
+      run: |
+        "${{ github.action_path }}" >> $Env:GITHUB_PATH
```

Comment:
This is duplicated below.

Reply:
Do you want me to still include the scripts in this new step, or include them at the beginning of the original `Setup` step?

Reply:
I think having just Setup Scripts like this here is better. But then also rename the below Setup step to something more describing.